osmaxx-py/osmaxx/utilities/shortcuts.py

```diff
@@ -0,0 +1,34 @@
+from django.shortcuts import _get_queryset
+
+
+def get_object_or_none(klass, *args, **kwargs):
+    """
+    Uses get() to return an object, or None if the object
+    does not exist.
+
+    klass may be a Model, Manager, or QuerySet object. All other passed
+    arguments and keyword arguments are used in the get() query.
+
+    Note: Like with get(), an MultipleObjectsReturned will be raised if more than one
+    object is found.
+    """
+    queryset = _get_queryset(klass)
+    try:
+        return queryset.get(*args, **kwargs)
+    except queryset.model.DoesNotExist:
```

Comment:
This is one place, where we don't want to do this. It is a utility function, if you use it, you really don't want to know about what happened, that's the actual purpose of this utility function.